cabot/rest_urls.py

```diff
@@ -124,33 +125,32 @@ class ViewSet(viewset_class):
     ),
 ))
 
-'''
-Omitting user API, could expose/allow modifying dangerous fields.
+# User API is off by default, could expose/allow modifying dangerous fields
+if settings.EXPOSE_USER_API:
+    router.register(r'users', create_viewset(
+        arg_model=django_models.User,
+        arg_fields=(
+            'password',
+            'is_active',
+            'groups',
+            #'user_permissions', # Doesn't work, removing for now
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+        ),
+    ))
 
-router.register(r'users', create_viewset(
-    arg_model=django_models.User,
-    arg_fields=(
-        'password',
-        'is_active',
-        'groups',
-        #'user_permissions', # Doesn't work, removing for now
-        'username',
-        'first_name',
-        'last_name',
-        'email',
-    ),
-))
+    router.register(r'user_profiles', create_viewset(
+        arg_model=models.UserProfile,
+        arg_fields=(
+            'user',
+            'mobile_number',
+            'hipchat_alias',
```

Comment:
Sorry, my point is that there's no point exposing the (legacy) `hipchat_alias` field because in fact the hipchat plugin doesn't read from there, rather from `HipchatAlertUserData.hipchat_alias`

So we might as well just remove this (and `mobile_number`)